__exekutir__/inventory/host_vars/kommandir.yml

```diff
@@ -87,4 +87,6 @@ git_cache_args:
     - repo: "https://github.com/python-bugzilla/python-bugzilla.git"
       recursive: False
       cachepath: "python-bugzilla"
+      # Do not include a 'depth' argument, git needs the full history to be 
+      # able to locate specific commits/branches/tags referenced by 'version'
```

Comment:
> No documentation in this module.

Ahh, yes.  Do you think it would be more useful there (in a comment) or in the proper Sphinx-based (pretty) docs?  (I was thinking of starting a "variables reference" section)